tooling/metamodel-generator/src/main/java/org/hibernate/processor/HibernateProcessor.java

```diff
@@ -253,9 +253,16 @@ private boolean handleSettings(ProcessingEnvironment environment) {
 		PackageElement quarkusOrmPanachePackage =
 				context.getProcessingEnvironment().getElementUtils()
 						.getPackageElement( "io.quarkus.hibernate.orm.panache" );
+		PackageElement quarkusPanache2Package =
+				context.getProcessingEnvironment().getElementUtils()
+						.getPackageElement( "io.quarkus.hibernate.panache" );
```

Comment:
We discussed at some point going with "Quarkus Data" instead of "Panache 2"... Are you sure you still want to go with this name and package? `io.quarkus.data.hibernate` could work too, in that case?

Maybe this is a discussion we should take offline.

Reply:
Did we? Or are you just trying to trick me because you know my memory doesn't go that far? 🤔